### PR TITLE
[1-0004: Create REST to get list of all restaurants]

### PR DESCRIPTION
**Pull request by Saparmyrat12**
Add integ test for repositoy layer

### CODE REVIEW

File: epam-vote-system-repository/src/testInteg/java/com/epam/vote/repository/testInteg/RestaurantRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+public class RestaurantRepositoryTest {
+
+    @Autowired
+    private IRestaurantRepository repository;
+
+    @Test
+    public void findAllRestaurants() {
+        List<Restaurant> restaurants = repository.findAllRestaurants();
+        Restaurant restaurant = restaurants.get(0);
+        assertNotNull(restaurants);
```
Comment (AzarenkaA) [inline epam-vote-system-repository/src/testInteg/java/com/epam/vote/repository/testInteg/RestaurantRepositoryTest.java:35]: redundant check

File: epam-vote-system-repository/src/testInteg/java/com/epam/vote/repository/testInteg/RestaurantRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        assertEquals(1, restaurants.size());
```
Comment (AzarenkaA) [inline epam-vote-system-repository/src/testInteg/java/com/epam/vote/repository/testInteg/RestaurantRepositoryTest.java:36]: check size before getting from collection

File: epam-vote-system-repository/src/testInteg/resources/test_IRestaurantRepository.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                            http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd
+                            http://www.liquibase.org/xml/ns/dbchangelog-ext
+                            http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet id="2021-05-06-2" author="sapar">
```
Comment (AzarenkaA) [inline epam-vote-system-repository/src/testInteg/resources/test_IRestaurantRepository.xml:11]: add comment for what test this data was added

File: epam-vote-system-repository/src/testInteg/java/com/epam/vote/repository/testInteg/EmbeddedPostgresConfig.java [hunk 4]
```diff
@@ -60,17 +56,26 @@ public SpringLiquibase liquibase() {
     public DataSource dataSource() {
         return new SimpleDriverDataSource(new Driver(), url);
     }
-/**
- * Change this property to your own that all tests will work
- */
+
+    /**
+     * Change this property to your own that all tests will work
+     */
```
Comment (AzarenkaA) [inline epam-vote-system-repository/src/testInteg/java/com/epam/vote/repository/testInteg/EmbeddedPostgresConfig.java:62]: remove this comment

File: epam-vote-system-repository/pom.xml [hunk 5]
```diff
@@ -48,6 +48,16 @@
             <artifactId>mybatis-spring</artifactId>
             <version>${mybatis.spring.version}</version>
         </dependency>
+        <dependency>
```
Comment (AzarenkaA) [inline epam-vote-system-repository/pom.xml:51]: what is the reason move this dependencies from service layer? revert it

Comment (Saparmyrat12) [reply epam-vote-system-repository/pom.xml:51]: slf4j asked dependencies

File: epam-vote-system-repository/src/testInteg/java/com/epam/vote/repository/testInteg/RestaurantRepositoryTest.java (hunk 2 above)
Comment (AntonAzarenka) [reply epam-vote-system-repository/src/testInteg/java/com/epam/vote/repository/testInteg/RestaurantRepositoryTest.java:35]: comment not resolved

File: epam-vote-system-repository/src/testInteg/resources/test_IRestaurantRepository.xml (hunk 3 above)
Comment (AntonAzarenka) [reply epam-vote-system-repository/src/testInteg/resources/test_IRestaurantRepository.xml:12]: comment not resolved

**Pull request merged**